ajax/libs/angular-toastr/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "angular-toastr",
+  "version": "1.6.0",
+  "author": {
+    "name": "Jesus Rodriguez",
+    "email": "Foxandxss@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Foxandxss/angular-toastr.git"
+  },
+  "filename": "angular-toastr.min.js",
+  "keywords": [
+    "angularjs",
+    "toastr",
+    "popup"
+  ],
+  "license": "MIT",
+  "description": "[![Code Climate](https://codeclimate.com/github/Foxandxss/angular-toastr.png)](https://codeclimate.com/github/Foxandxss/angular-toastr) [![Build Status](https://travis-ci.org/Foxandxss/angular-toastr.svg?branch=master)](https://travis-ci.org/Foxandxss/ang",
```

Comment:
This description looks strange, should not use markdown here, please update it to a normal form.